docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -392,10 +392,20 @@ These roles are then applicable for endpoint authorization by using the @RolesAl
 [source,properties]
 ----
 quarkus.http.auth.policy.admin-policy1.roles.admin=Admin1 <1>
-quarkus.http.auth.permission.roles1.paths=/*
+quarkus.http.auth.permission.roles1.paths=/* <2>
 quarkus.http.auth.permission.roles1.policy=admin-policy1
 ----
 <1> Map the `admin` role to `Admin1` role. The `SecurityIdentity` will have both `admin` and `Admin1` roles.
+<2> The `/*` path is secured, only authenticated HTTP requests are granted access.
+
+If all that you need is to map the `SecurityIdentity` roles to the deployment-specific roles regardless of a path, you can also do this:
+
+[source,properties]
+----
+quarkus.http.auth.roles-mapping.admin=Admin1 <1>
```

Comment:
It looks fine, but from the earlier discussion related to improving sharing roles, you were thinking about the `quarkus.security` namespace. So should we do it here too ?
I don't mind to using the `http.auth` though for now.

Reply:
As at the moment we don't have a clear architecture for non http transport, may be for GRPC but not messaging

Reply:
> As at the moment we don't have a clear architecture for non http transport, may be for GRPC but not messaging

My understanding is that gRPC basically runs on the top of HTTP/2 and that soon there will be switch in default that will result in gRPC running on the same server (just as a root) as our other root handlers (=just another route handler,like RR and RC).

Reply:
Meaning this is alright for a gRPC, it will work as we already support HTTP Security Policies when gRPC runs on the same server as other route handlers.

Reply:
@michalvavrik Hi, 
>I went with the http.auth as you suggested we need this aligned with HTTP permissions. 

I'm not sure I understand, why do we have to align it with HTTP permissions ?

Reply:
citing your sentence from https://github.com/quarkusio/quarkus/issues/39269#issuecomment-1984256174 :

> But we have to align it with the HTTP policy way too, what happens if both quarkus.security.root.roles and quarkus.http.auth.permission.roles1.shared=true is set...

That is why I rewrote it. Maybe I simply misunderstood it.

This PR is based on this suggestion: https://github.com/quarkusio/quarkus/issues/37989#issuecomment-1875780217 were you proposed:

```
quarkus.http.auth.roles-mapping.administrator=admin
quarkus.http.auth.roles-mapping.customer=user
```

Apart of that, I can provide you my view - we document here `SecurityIdentity` roles mapping https://quarkus.io/guides/security-authorize-web-endpoints-reference#map-security-identity-roles and this I meant to complement this path-specific mapping for all the paths.

Anyway, I won't pretend I can start from scratch and move it to the Security extension if you think it should be there :-)

Reply:
Ha! Now thinking of it, this implementation has also one advantage - you can provide separate mapping for management port and I can imagine there can be different need for roles.

Reply:
@michalvavrik 
>you can provide separate mapping for management interface

Yeah it is a good one. And similarly for shared roles (I still have to open the issue)